### DISCUSSION TITLE
Octokit.Net and GraphQL

### DISCUSSION THREAD

**Discussion by Gregory-Lange** [octokit/.github#41] [General]
So I know you have an Octokit.graphql.net but it appears most files haven't been touched in a while, 3 or more months old, where Octokit.Net has more recent commits more between 2 to 7 months.  Is the octokit.graphql.net usable or does it need work?  The reason i ask is I am impressed with GraphQL and just wasn't sure what that repo had in it compared to octokit.net.

**Comment by Gregory-Lange** [comment]
Was wondering if anyone had an answer for this?  Did the GraphQL just get rolled into Octokit.Net, but i don't see calls like the javascript one where you call GraphQL method and pass in a query with option interface.  Or do we need to just pull the Octokit.GraphQL.Net and update it to latest C#/.Net Standard/etc. ?  Again I really like GraphQL has to offer but wasn't sure.

**Comment by cliffchapmanrbx** [comment]
FWIW we're actively using this in internal-critical apps and it's working fine for the most part. Generally code changes are only necessary when adding a new API that doesn't currently exist.
